presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -439,6 +441,31 @@ public long getOutputPositions()
     }
 
     @JsonProperty
+    public long getWrittenPositions()
+    {
+        // Using the input stats of the operator.
```

Comment:
I like that you have a comment, but this code is pretty self-explanatory for anybody who has spent time working on Presto. Let's remove it so that there's less chance of comment diverging from code in the future.